code/game/machinery/navbeacon.dm

```diff
@@ -49,13 +60,16 @@
 		GLOB.navbeacons["[new_z]"] += src
 	..()
 
+//Monkestation edit start
```

Comment:
Just put this next to the change you made instead of surrounded some of the code you didn't touch